### ISSUE TITLE
"anastasia" not working in server at Russia Map

### ISSUE THREAD

**Issue by GBLKai** [SmartlyDressedGames/U3-Docs#62]
I used U3DS start the server without Rocket or Openmod,even workshop.
Commands.dat only "map russia"
Config.json "UnityEvents.Allow_Client_Messages" and "UnityEvents.Allow_Client_Commands" are "true"
"night""area" be noticed
It worked in single,but it not working in server for some reason.

**Comment by SDGNelson** [comment]
In order to double-check this I loaded up a vanilla server on Russia and was able to open the door as-expected. There are a lot of variables at play so I am not sure what specifically happened in your case, but I am confident this works properly.

**Comment by GBLKai** [comment]
![20210526001459_1](https://user-images.githubusercontent.com/84732729/119532669-aaf85b80-bdb7-11eb-924a-98aebef6181d.jpg)
OK,I reinstalled the server.
It just not worked.
I think it's probably some options was wrong.
I google and go reddit try to find some solutions.but nothing.
So could you tell me which variables or options or course can cause or solve this ?
I'm appreciated.

**Comment by GBLKai** [comment]
By the way.
This server's physical location and ip address are China.
This may cause problem?

**Comment by SDGNelson** [comment]
To double-check that it is nighttime you can use the /night command.

Is it an edited copy of the Russia map? If the file name is not "Russia" then some of the vanilla Russia quest stuff is not loaded.

**Comment by GBLKai** [comment]
yes,it's night for sure.
I will post the logs
[2021-05-26 00:12:32] Successfully set time to night!
[2021-05-26 00:13:34] Successfully gave Kai_Kay 1365 x1!
[2021-05-26 00:14:19] [Area] Kai_Kay [Kai_Kay]: "anastasia"
[2021-05-26 00:14:22] [Area] Kai_Kay [Kai_Kay]: "Anastasia"
[2021-05-26 00:14:25] [Area] Kai_Kay [Kai_Kay]: "anastasia"
[2021-05-26 00:14:27] [Area] Kai_Kay [Kai_Kay]: "anastasia"
[2021-05-26 00:14:28] [Area] Kai_Kay [Kai_Kay]: "anastasia"
[2021-05-26 00:14:29] [Area] Kai_Kay [Kai_Kay]: "anastasia"
[2021-05-26 00:14:35] [Area] Kai_Kay [Kai_Kay]: "anastasia"
[2021-05-26 00:14:49] [Area] Kai_Kay [Kai_Kay]: "Anastasia"
[2021-05-26 00:14:53] [Area] Kai_Kay [Kai_Kay]: "anastasia"
[2021-05-26 00:14:54] [Area] Kai_Kay [Kai_Kay]: "Anastasia"
[2021-05-26 00:14:55] [Area] Kai_Kay [Kai_Kay]: "Anastasia"
[2021-05-26 00:14:56] [Area] Kai_Kay [Kai_Kay]: "Anastasia"
[2021-05-26 00:14:59] [Area] Kai_Kay [Kai_Kay]: "anastasia"
[2021-05-26 00:15:28] Successfully admined 76561198141351890!
[2021-05-26 00:15:32] [Area] Kai_Kay [Kai_Kay]: "@night"
[2021-05-26 00:15:32] Successfully set time to night!
[2021-05-26 00:15:37] [Area] Kai_Kay [Kai_Kay]: "anastasia"
[2021-05-26 00:15:38] [Area] Kai_Kay [Kai_Kay]: "anastasia"
[2021-05-26 00:15:39] [Area] Kai_Kay [Kai_Kay]: "anastasia"
[2021-05-26 00:15:45] [Area] Kai_Kay [Kai_Kay]: "@night"
[2021-05-26 00:15:45] Successfully set time to night!
[2021-05-26 00:15:48] [Area] Kai_Kay [Kai_Kay]: "@night"
[2021-05-26 00:15:48] Successfully set time to night!

and in "Commands.dat"
map russia
here is logs again

[2021-05-26 00:10:55] Successfully set map to russia!
[2021-05-26 00:10:55] Successfully set gamemode to hard!

The files
[Server_Server1.log](https://github.com/SmartlyDressedGames/U3-Docs/files/6542561/Server_Server1.log)
[Server_Server1_Prev.log](https://github.com/SmartlyDressedGames/U3-Docs/files/6542562/Server_Server1_Prev.log)

**Comment by GBLKai** [comment]
and the server it self
`start %~dp0ServerHelper.bat +InternetServer/Server1`
not lan server
i can search in the public server list

**Comment by SDGNelson** [comment]
Ah: if you change the command from "Map russia" to "Map Russia" it should work. Sorry about this. Some of these older features are case sensitive, so I will get that fixed for the next update.

**Comment by GBLKai** [comment]
and no any workshop files
`{
  "File_IDs": [],
  "Ignore_Children_File_IDs": [],
  "Query_Cache_Max_Age_Seconds": 600,
  "Max_Query_Retries": 2,
  "Use_Cached_Downloads": true,
  "Should_Monitor_Updates": true,
  "Shutdown_Update_Detected_Timer": 600,
  "Shutdown_Update_Detected_Message": "Workshop file update detected, shutdown in: {0}",
  "Shutdown_Kick_Message": "Shutdown for Workshop file update."
}`
if it can load edited copy without workshop,well i don't know that.
and the NPC worked fine in the aircraft carrier.
i even finished "On air" and "Research Recovery"
the "Bones" was found correctly in tank and collapsed subway station.
i direct teleport to the laboratory,and successfully opened the door need key before reinstalled server by Rocket.
but couldn't use the "red stone" teleport back,so i give up.
and i direct teleport behind "Anastasia"'s door,and kill the zombies.
i think it not work,i not saw blood particle effects,in single i saw the effects.

**Comment by GBLKai** [comment]
Ok sorry let me try again.

**Comment by GBLKai** [comment]
Finally!!
It's worked without reinstall server.
I just change `russia` to `Russia`
Maybe "Hawaii" had same issues?
I'm not try yet.
but
Thank you very much for your help, it is very helpful to me!!
Hope everything goes well for you!!!

**Comment by SDGNelson** [comment]
Glad to hear that did the trick! Sorry for the trouble. Closing again now because I have fixed this for the Friday update this week.
